ovos_bus_client/client/client.py

```diff
@@ -184,7 +189,7 @@ def emit(self, message: Message):
         if hasattr(message, 'serialize'):
             msg = message.serialize()
         else:
-            msg = json.dumps(message.__dict__)
+            msg = orjson.dumps(message.__dict__).decode("utf-8")
```

Comment:
Note for the future: we should test if we can remove the `.decode()` call here. It's in place to force the return value to be a string instead of bytes, for consistency, but bytes might be fine. Would require extensive testing in a whole system to be sure.